test/nsidc-icesat2/NSIDC-ICESAT2_Regression.ipynb

```diff
@@ -220,10 +220,107 @@
     "            'shape': 'ancillary/Ross_Sea_positive_lon_only.geojson',\n",
     "        },\n",
     "    },\n",
-    "}\n",
-    "\n",
-    "\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34aa51c3-39d7-46c8-ade2-4ae771452798",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "production_configuration = {\n",
+    "    'subset_bounding_box': {\n",
+    "        'ATL03': {\n",
+    "            'collection_concept_id': Collection(id='C2596864127-NSIDC_CPRD'),\n",
+    "            'granule_id': 'G2632805836-NSIDC_CPRD',\n",
+    "            'spatial': BBox(-105.5, 40.0, -105.0, 40.005),\n",
+    "        },\n",
+    "        'ATL07': {\n",
+    "            'collection_concept_id': Collection(id='C2713030505-NSIDC_CPRD'),\n",
+    "            'granule_id': 'G2738665484-NSIDC_CPRD',\n",
+    "            'spatial': BBox(-112.0, 80.0, -93.0, 80.3),\n",
+    "        },\n",
+    "        'ATL08': {\n",
+    "            'collection_concept_id': Collection(id='C2613553260-NSIDC_CPRD'),\n",
+    "            'granule_id': 'G2738665484-NSIDC_CPRD',\n",
```

Comment:
I think this might be a copy and paste error - it's the same granule concept ID as used for ATL07.